src/components/gitOps/gitops.service.tsx

```diff
@@ -24,4 +24,9 @@ export function getGitOpsConfigurationList(): Promise<any> {
 export const validateGitOpsConfiguration = (request: any):Promise<any> => {
   const URL = `${Routes.GITOPS_VALIDATE}`;
   return post(URL, request);
-}
\ No newline at end of file
+}
+
+export const validateHelmAppGitOpsConfiguration = (request: any): Promise<any> => {
+  const URL = `${Routes.GITOPOS_HELM_VALIDATE}`
+  return post(URL, request);
```

Comment:
We can directly use `Routes.GITOPOS_HELM_VALIDATE` here. no need to create variable